asset-swapper: isBlacklisted in rfqtMakerInteraction

feuGeneA opened the pull request.

feuGeneA committed:
asset-swapp: isBlacklisted in rfqtMakerInteraction

PirosB3 commented on packages/asset-swapper/src/utils/quote_requestor.ts:
```diff
@@ -337,10 +337,9 @@ export class QuoteRequestor {
-                    !rfqMakerBlacklist.isMakerBlacklisted(url)
-                ) {
+                if (rfqMakerBlacklist.isMakerBlacklisted(url)) {
+                    this._infoLogger({ rfqtMakerInteraction: { url, quoteType, isBlacklisted: true } });
```
can you also add `requestParams`?

PirosB3 reviewed (approved):
Aside one comment, it's good to go!

feuGeneA committed:
Include requestParams in maker-blacklisted log

Addresses review comment https://github.com/0xProject/0x-monorepo/pull/2726#discussion_r501332160
